dev/otel-agent-config.yaml

```diff
@@ -1,16 +1,16 @@
 receivers:
-  jaeger:
+  otlp:
     protocols:
-      thrift_compact: # on port 6831
-      thrift_binary: # on port 6832
+      http:
```

Comment:
This is empty, does it work?

Reply:
Yup I think so

Reply:
Worked locally

Reply:
Okay, just saw it's enabled by default in the otel chart so don't need to explicitly enable it in deployments.